openlibrary/plugins/worksearch/subjects.py

```diff
@@ -259,7 +252,7 @@ def create_engine():
     subject_results = engine.get_subject(
         key, details=details, offset=offset, sort=sort_order,
         limit=limit, **filters)
-    return inject_availability(subject_results)
+    return subject_results
```

Comment:
This seems like it might change a lot of things. Can you confirm it's ok?

Reply:
Yes -- in the line above (`engine.get_subject`) we're adding `add_availability` directly to these works